docs/reference/relay-types.md

```diff
@@ -2,9 +2,26 @@
 
 Relay's workflow dialect uses YAML "tag" notation, indicated by a `!`, to identify custom data types that the Relay service associates with code that it should run the workflow is executing. This allows workflows to have dynamic values, instead of hard coding everything directly in the YAML.  There are several top-level types described here, plus a set of data-manipulation functions accessed as `!Fn.<function>` which are [documented in the function reference](relay-functions.md).
 
+## !Connection
+
+Most actions require some form of authentication. `!Connection` provides a way to add service credentials to the workflow that are required by a Relay action. Actions that require a `!Connection` in order to run can be configured as follows: 
+
+```yaml
+- name: describe-instances
+  image: projectnebula/ec2-describe-instances
+  spec:
+    aws: 
+      connection: !Connection { type: aws, name: my-aws-account  } 
```

Comment:
is there a predefined list of 'types' that Connections can be? if so it'd be good to include that